core/primitives/src/hasher.rs

```diff
@@ -42,6 +42,13 @@ pub mod blake2 {
 	#[derive(Debug)]
 	pub struct Blake2Hasher;
 
+	impl Blake2Hasher {
```

Comment:
What's wrong with `use Hasher`?

Reply:
It's only in `hash_db` crate - this was a temporary, quick and dirty way to move forward. I now changed that and `Hasher` trait is re-imported from `primitives`, do you feel it's ok?